test_scan.py

```diff
@@ -222,5 +222,42 @@ def test_sprint2_t3_docx(self):
         self.assertIn('John Smith', pii_text[0])
 
 
+    def test_sprint2_t4_xlsx(self):
```

Comment:
I'm OK with what you've done here but I don't think you should name your test case _xlsx since you are testing multiple files types.  perhaps test_sprint2_t4 is most appropriate name?